app/views/idv/doc_auth/welcome.html.erb

```diff
@@ -97,8 +108,8 @@
         ].select(&:present?),
       ) %>
 
-  <h2><%= t('doc_auth.instructions.privacy') %></h2>
-  <p>
+  <h3><%= t('doc_auth.instructions.privacy') %></h3>
+  <p class="padding-bottom-2">
```

Comment:
Poking around some of the other flow designs, I wonder if we could update the "Cancel" partial to have `margin-top-4` to achieve this same effect?

cc @anniehirshman-gsa do you think this makes sense? In other words, using 2rem top margin wherever we have "Cancel" link at the bottom of a page, where currently it is 1rem by default.

Reply:
Yeah, that definitely crossed my mind but I also didn't know what the impact would be of that across all the places it is used. It seemed like this was mostly because of the lack of a "start over" to go with it so I figured it could just be a one off. But if it's safe that's fine, I'll change it.

Reply:
> I wonder if we could update the "Cancel" partial to have `margin-top-4` to achieve this same effect?

Hm, one challenge is that the [Start Over or Cancel partial](https://github.com/18F/identity-idp/blob/main/app/views/idv/doc_auth/_start_over_or_cancel.html.erb) composes the "Cancel" link, where we'd not want `2rem` between "Start Over" and the Cancel link's border 🤔

Reply:
+ a consistent `2 rem` space above the start over/cancel/back component would be great, but not sure how to best achieve that or if it's possible to make that change more widely without creating spacing issues elsewhere

Reply:
Yeah, I worry adding it globally here would create some oddities elsewhere that'll be a bit hard to nail down quickly.

Reply:
Given the limited scope of this ticket and change set it would make more sense to split off the global change of styling for the start over/cancel/back component to another ticket.